### PR TITLE
Metrics update with examples

### PR DESCRIPTION
**Pull request by hoh**
This updates the documentation on metrics and adds examples on how to fetch metrics.

New code example is tested using `pytest`.

image

### CODE REVIEW

File: docs/nodes/reliability/metrics.md [hunk 1]
```diff
@@ -59,13 +92,63 @@ The metrics for a CRN have the following form
...
+The [scores](../../nodes/reliability/scores.md) are computed based on the metrics, in a reproducible manner.
+
+Metrics messages can be found:
+
+### On the Message Explorer
+
+[https://explorer.aleph.im/messages?showAdvancedFilters=1&channels=aleph-scoring&sender=0x4D52380D3191274a04846c89c069E6C3F2Ed94e4](https://explorer.aleph.im/messages?showAdvancedFilters=1&channels=aleph-scoring&sender=0x4D52380D3191274a04846c89c069E6C3F2Ed94e4)
+
+### Using the Python SDK
+
```
Comment (1yam) [inline docs/nodes/reliability/metrics.md:111]: Could maybe here add the link to the aleph-sdk-python doc ?

**Review by 1yam — APPROVED** [review]
Else LGTM

File: docs/nodes/reliability/metrics.md (hunk 1 above)
Comment (hoh) [inline docs/nodes/reliability/metrics.md:111]:
```suggestion

The [Python SDK](../../libraries/python-sdk/posts/query/) provides helpers to fetch the relevant messages.
```

File: docs/nodes/reliability/metrics.md [hunk 2]
```diff
@@ -59,13 +92,64 @@ The metrics for a CRN have the following form
...
+The [Python SDK](../../libraries/python-sdk/posts/query/) provides helpers to fetch the relevant messages.
```
Comment (hoh) [inline docs/nodes/reliability/metrics.md:112]:
```suggestion
The [Python SDK](../../../libraries/python-sdk/posts/query/) provides helpers to fetch the relevant messages.
```

File: docs/nodes/reliability/metrics.md (hunk 1 above)
Comment (hoh) [reply docs/nodes/reliability/metrics.md:111]: Fixed

**Pull request merged**